Resources/Prototypes/Entities/Structures/Walls/asteroid.yml

```diff
@@ -7,6 +7,25 @@
   - type: RandomAppearance
     key: enum.AsteroidRockVisuals.State
   - type: Mineable
+    ores:
+    - id: SteelOre1
+      prob: 0.25
+      orGroup: Asteroid
+    - id: GoldOre1
+      prob: 0.05
+      orGroup: Asteroid
+    - id: SpaceCrystal1
+      prob: 0.10
+      orGroup: Asteroid
+    - id: PlasmaOre1
+      prob: 0.05
+      orGroup: Asteroid
+    - id: SilverOre1
+      prob: 0.025
+      orGroup: Asteroid
+    - id: UraniumOre1
+      prob: 0.025
+      orGroup: Asteroid
```

Comment:
Long-term there should be different rock types to represent veins rather than prioritising bulk mining.

Reply:
Yup I was thinking later on, we would have a vein generator for mappers to use. Having asteroid rocks be the source is only temporary